dbux-code/src/traceDetailsView/nodes/ValueTDSimpleNode.js

```diff
@@ -78,6 +79,7 @@ export default class ValueTDSimpleNode extends ValueNode {
   }
 
   valueRender() {
+    emitValueRenderAction(this.value, this.nodeId);
```

Comment:
@MiccWan 
Good rule of thumb: either place all events **before** or place all events **after** the thing happened. Don't ever switch orders on that. Since we are mostly calling `emit` afterwards, let's try to make sure, we keep that order